### ISSUE TITLE
Electron desktop app

### ISSUE THREAD

**Issue by ajvsol** [etesync/etesync-web#31]
It would be great to have a desktop app available and it shouldn't be too much work to create one if using Electron.

**Comment by tasn** [comment]
I agree, that could be cool. Do you have experience in doing that? As in, do you know how big of an hassle it is?

**Comment by tasn** [comment]
Thanks for the link. I'll try to get to it at some point, though it's a bit of a lower priority than others.

For anyone reading this: I'll happily accept a PR doing it.
